### PR TITLE
Enable users to run spacestation and groundstation directly from Docker

### PR DESCRIPTION
**Pull request by bigtallcampbell**
Setup to run as a single file on both devContainers and on a blank VM
Moved sync folders to be more contextually intuitive - toSpaceStation / fromSpaceStation and toGroundStation / fromGroundStation
Changed devcontainer docker instance to use embedded docker
Updated output instructions to be shown on the screen

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -28,19 +27,12 @@
...
-	"updateContentCommand": "/tmp/library-scripts/mock-spacestation-setup.sh",
-	"mounts": [
-		"source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind",
-		"source=${env:HOME}${env:USERPROFILE}/.ssh,target=/home/vscode/.ssh,type=bind"
-	],
-	"remoteUser": "vscode",
+	"updateContentCommand": ". /groundstation/deployGroundStation.sh",
+	// "mounts": [
+	// 	//"source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind"
+	// ],
```
Comment (glennmusa) [inline .devcontainer/devcontainer.json:33]: if we don't need it any more let's delete it
```suggestion
```

File: .devcontainer/setupScripts/Dockerfile.SpaceStation [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        && sed -i'' -e's/^#PermitEmptyPasswords no$/PermitEmptyPasswords yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^UsePAM yes/UsePAM no/' /etc/ssh/sshd_config && service ssh start
+
+
+# Start up the SSH Server Daemon and the docker-in-docker wrapper script
+CMD ["sh", "-c", "/usr/sbin/sshd ; /bin/bash"]
+
+
+# USER $SPACESTATION_USER
+#WORKDIR $GROUNDSTATION_ROOTDIR
```
Comment (glennmusa) [inline .devcontainer/setupScripts/Dockerfile.SpaceStation:70]: if we don't need it let's remove it

File: .devcontainer/setupScripts/deployGroundStation.sh [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+# ********************************************************
+# Miscellaneous Directories: END
+# ********************************************************
+
+
+# ********************************************************
+# Persistant Variables: START
+# ********************************************************
+#sudo bash -c 'cat > /etc/profile.d/mock-spacestation-vars.sh' << EOF
```
Comment (glennmusa) [inline .devcontainer/setupScripts/deployGroundStation.sh:44]: if we don't need it let's remove it

**Review by glennmusa — CHANGES_REQUESTED** [review]

File: .devcontainer/setupScripts/runFromAzureTemplate.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
+# Used on a blank VM Sets via Azure templates to deploy the Mock SpaceStation environment.  Mostly a wrapper for deployGroundStation.sh
+# Syntax: ./runFromAzureTemplate.sh
+
+curl -O "https://raw.githubusercontent.com/azure/mock-spacestation/.devcontainer/setupScripts/deployGroundStation.sh"
+. ./deployGroundStation.sh
```
Comment (glennmusa) [inline .devcontainer/setupScripts/runFromAzureTemplate.sh:11]: the repo is lightweight enough to clone in its entirety no? this way we don't need to have those encoded strings in a script.

Comment (glennmusa) [reply .devcontainer/setupScripts/runFromAzureTemplate.sh:11]: or, if we're married to this idea, let's curl the files in `deployFromGroundstation`

Comment (bigtallcampbell) [reply .devcontainer/setupScripts/runFromAzureTemplate.sh:11]: This is just a wrapper for the Azure Template so we can change/update the deployGroundStation.sh and not affect the template initiation / json file.  More-or-less removes a step in future updates.

**Pull request closed without merging**